app/src/main/AndroidManifest.xml

```diff
@@ -94,6 +94,14 @@
                 android:value="org.fossasia.susi.ai.login.LoginActivity" />
         </activity>
 
+        <activity android:name=".device.managedevices.ManageDeviceActivity"
```

Comment:
I prefer a fragment instead

Reply:
We will have fragments inside this activity as we go on implementing the features here. It will act as the base for those fragments.

Reply:
@liveHarshit please review it now